ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/BaseCommand.java

```diff
@@ -45,4 +48,14 @@
   )
   protected boolean dryRun = false;
 
+  protected static final Logger LOGGER = LoggerFactory.getLogger(Migrations.class);
+
+  @Override
+  public void run() {
+    final long startTime = System.nanoTime();
+    command();
+    LOGGER.info("Execution time: " + (System.nanoTime() - startTime) / 1000000000);
```

Comment:
I think it's better to have the different classes each have their own loggers, rather than having them all share the `Migrations.class` logger. By having different loggers, users can more easily filter logs and can also configure different log levels for the different loggers.

I assume your motivation for switching to a shared logger was so that the execution time would be logged by the same logger as the other messages from the command execution itself? I don't have a great solution but one option could be to have each command implement a simple `getLogger()` method, which we can then call here.